modules/ossm-creating-certificates-for-multi-cluster-mesh.adoc

```diff
@@ -0,0 +1,208 @@
+// This module is used in the following assemblies:
+
+// * install/ossm-multi-cluster-topology.adoc
+
+:_mod-docs-content-type: Procedure
+[id="ossm-creating-certificates-for-a-multi-cluster-mesh_{context}"]
+= Creating certificates for a multi-cluster mesh 
+
+Create the root and intermediate certificate authority (CA) certificates for two clusters.
+
+.Prerequisites
+
+* You have OpenSSL installed locally.
+
+.Procedure
+
+. Create the root CA certificate.
+
+.. Create an OpenSSL configuration certificate file named `root-ca.conf` for the root CA certificates:
```

Comment:
Flip this step and step "b. Create a key for the root certificate...", so that the order matches the order in "2. Create the intermediate CA certificate for the East cluster" and "3. Create the intermediate CA certificate for the West cluster".

Reply:
done